Makefile

```diff
@@ -37,13 +37,18 @@ protos: ## Generate go code from .proto files
 	@docker build . --target protos \
 	--output ./cli/server/protos
 
-cli: ## Compile the cli
+cli: completion ## Compile the cli
 	@docker build . --target cli \
 	--platform local \
 	--build-arg BUILD_TAGS=e2e,kube \
 	--build-arg GIT_TAG=$(GIT_TAG) \
 	--output ./bin
 
+completion: ## Compile completion binary to generate completion scripts
+	@docker build . --target completion \
+	--platform local \
```

Comment:
for completion generation, --platform is useless

Reply:
I actually realized that building the completion binary is not necessary.
This artifact is of no servitude. A simple `go run` to generate the completions is the simplest  solution to this avoiding the whole docker build and platform related things.